GitUI/AutoCompletion/CommitAutoCompleteProvider.cs

```diff
@@ -121,7 +121,8 @@ private static string GetChangedFileText (GitModule module, GitItemStatus file)
             // Try to read the contents of the file: if it cannot be read, skip the operation silently.
             try
             {
-                return File.ReadAllText(Path.Combine(module.WorkingDir, file.Name));
+                IFullPathResolver _fullPathResolver = new FullPathResolver(() => module.WorkingDir);
+                return File.ReadAllText(_fullPathResolver.Resolve(file.Name));
```

Comment:
This particular change doesn't feel like making the intent any clearer, nor the current implementation lends itself to a good refactor (I spent some time tinkering with it) without a major work.
Suggest rolling back the change here and let it remain in its current state. Eventually we may get around refactoring it.

Reply:
Reason to change was to set the pattern, but this was not obvious. Reverting.